flickr.py

```diff
@@ -25,16 +25,17 @@ def params_unique_combination(baseurl, params_d, private_keys=["api_key"]):
             res.append("{}-{}".format(k, params_d[k]))
     return baseurl + "_".join(res)
 
-def search_flickr_by_tags(tags):
+def search_flickr(params_dict):
     if not FLICKR_API_KEY:
         raise Exception('Flickr API Key is missing!')
 
     baseurl = "https://api.flickr.com/services/rest/"
     params_diction = {
-        "method": "flickr.photos.search",
+        "method": "flickr.photos." + params_dict["method"],
+        "photo_id": params_dict["photo_id"],
         "format": "json",
         "api_key": FLICKR_API_KEY,
-        "tags": tags,
+        "tags": params_dict["tags"],
```

Comment:
Do all of the API calls require a "tags" parameter?

Reply:
No, it's not supposed to, but I know I did not get this to successfully work.